test/lib/rules/blank-lines/blank-lines.js

```diff
@@ -330,3 +330,150 @@ describe("[RULE] blank-lines: Rejections", function() {
     });
 
 });
+
+
+describe("[RULE] blank-lines: Fixes", function() {
```

Comment:
use `() => {` in place of `function() {`. In general, use all the ES6 features you can except for import statements. Solium was originally written in es5 so you will see es5-style code at a lot of places but we're now encouraging es6!